packages/@aws-cdk/aws-backup/lib/plan.ts

```diff
@@ -125,6 +126,7 @@ export class BackupPlan extends Resource implements IBackupPlan {
     const plan = new CfnBackupPlan(this, 'Resource', {
       backupPlan: {
         backupPlanName: props.backupPlanName || id,
+        backupOptions: props.backupOptions || {},
```

Comment:
This is not the correct location for this value, please consult [the documentation for this resource](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-backup.CfnBackupPlan.html).